coderd/notifications/notifications_test.go

```diff
@@ -762,7 +771,9 @@ func TestNotificationTemplates_Golden(t *testing.T) {
 			name: "TemplateUserAccountCreated",
 			id:   notifications.TemplateUserAccountCreated,
 			payload: types.MessagePayload{
-				UserName: "Bobby",
+				UserName:     "Bobby",
+				UserEmail:    "bobby@coder.com",
+				UserUsername: "bobby",
 				Labels: map[string]string{
 					"created_account_name":      "bobby",
 					"created_account_user_name": "William Tables",
```

Comment:
Note to self: Given `UserName` here and `created_account_user_name` a few lines below. Do I need both?